Kong/docs.konghq.com
# [DOCU-3024] Runtime manager IA

### Description

Roll-up of all the runtime manager IA work, rebased and ready to go.

https://konghq.atlassian.net/browse/DOCU-3024

### Testing instructions

Netlify link: 

❗ Reviewers: 
* Please test everything carefully to make sure we have no broken content.
* Review the section wholistically: does it all still make sense, now that we've reorganized it? Are we missing anything important?


### Checklist
